forge-gui/res/cardsfolder/upcoming/loot_dispute.txt

```diff
@@ -0,0 +1,11 @@
+Name:Loot Dispute
+ManaCost:3 R
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigInitiative | TriggerDescription$ When CARDNAME enters the battlefield, you take the initiative and create a Treasure token.
+SVar:TrigInitiative:DB$ TakeInitiative | SubAbility$ DBTreasure
+SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac
+T:Mode$ AttackersDeclaredOneTarget | ValidAttackers$ Creature.YouCtrl | AttackedTarget$ Player.hasInitiative | Execute$ DBTreasure | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack the player who has the initiative, create a Treasure token.
+T:Mode$ DungeonCompleted | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you complete a dungeon, create a 5/5 red Dragon creature token with flying.
+SVar:TrigToken:DB$ Token | TokenScript$ r_5_5_dragon_flying
+DeckHas:Ability$LifeGain|Token|Sacrifice & Type$Artifact|Treasure|Dragon
```

Comment:
Where does LifeGain come from? Can't quite see the ability that causes it...

Reply:
heh, from copy/pasting 😅